src/SFA.DAS.AssessorService.Web/Views/Certificate/Check.cshtml

```diff
@@ -6,6 +6,7 @@
 
 @{
     ViewBag.Title = (!ViewData.ModelState.IsValid ? "Error: " : "") + "Check and approve the assessment details";
+    Model.StandardHasOptions = (bool)TempData["HideOption"] == false;
```

Comment:
I would rework this. Rather than copy the temp data into the model, just use the model.

`            var sessionString = SessionService.Get(nameof(CertificateSession));
            if (sessionString == null)
            {
                return RedirectToAction("Index", "Search");
            }
            var certSession = JsonConvert.DeserializeObject<CertificateSession>(sessionString);
            TempData["HideOption"] = certSession.Options == null || !certSession.Options.Any();
            TempData["HideChangeOption"] = certSession.Options == null || certSession.Options.Count <= 1;
            TempData["HideChangeVersion"] = certSession.Versions == null || certSession.Versions.Count <= 1;`

Migrate the above into the LoadViewModel method, as that already also retrieves the session. Create Model properties for all 3 of the above ( so 2 more than what you have ) then just set that on the model rather than temp data, and switch the view to use the model properties. 

Feels like it would simplify the overall approach?